src/Gren/Kernel/Debug.js

```diff
@@ -141,6 +141,13 @@ function _Debug_toAnsiString(ansi, value) {
     return _Debug_internalColor(ansi, "<" + value.name + ">");
   }
 
+  if (
+    typeof _Array_Builder !== "undefined" &&
+    value instanceof _Array_Builder
+  ) {
+    return _Debug_internalColor(ansi, "<internals>");
```

Comment:
Why not show the array contents? (Maybe wrapped with ArrayBuilder, maybe not)

Reply:
Showing the contents make the mutation observable.

Reply:
Oh good point. And even better point about copying the array 😄